### PR TITLE
Enable travis

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  #- test: --platform vm,chrome
+  - dartanalyzer: --fatal-infos --fatal-warnings .
```
Comment (natebosch) [inline .travis.yml]: this task doesn't have a nice label in Travis. Maybe we don't care though?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  #- test: --platform vm,chrome
```
Comment (natebosch) [inline .travis.yml]: why aren't we running tests?

File: bin/dhttpd.dart [hunk 3]
```diff
@@ -1,3 +1,40 @@
-import '_boot.dart';
+import 'dart:io';
```
Comment (natebosch) [inline bin/dhttpd.dart]: [nit] blank line between `dart:` and `package:` imports

File: bin/dhttpd.dart [hunk 4]
```diff
@@ -1,3 +1,40 @@
...
+        defaultsTo: DEFAULT_HOST.toString(),
+        valueHelp: 'host',
+        help: 'The hostname to listen on (defaults to "localhost").')
+    ..addFlag('help', abbr: 'h', negatable: false, help: 'Displays the help.');
+
+  var results = argParser.parse(args);
+
+  if (results['help']) {
+    print(argParser.usage);
+    exit(1);
```
Comment (natebosch) [inline bin/dhttpd.dart]: [nit] I usually expect asking for help to not be considered a failure

File: bin/dhttpd.dart [hunk 5]
```diff
@@ -1,3 +1,40 @@
...
+
+  var port = int.parse(results['port'], onError: (source) {
+    stderr.writeln('port must be a number');
+    exit(1);
+  });
+
+  var hostname = results['host'];
+
+  String path =
+      results['path'] != null ? results['path'] : Directory.current.path;
```
Comment (natebosch) [inline bin/dhttpd.dart]: [nit] use `??`. Also consider using `var`

```dart
var path = results['path'] ?? Directory.current.path;
```

File: .travis.yml (hunk 2 above)
Comment (kevmoo) [reply .travis.yml]: There are none. 😄 TODO

File: bin/dhttpd.dart (hunk 3 above)
Comment (kevmoo) [reply bin/dhttpd.dart]: ack

File: bin/dhttpd.dart (hunk 4 above)
Comment (kevmoo) [reply bin/dhttpd.dart]: ack...

File: bin/dhttpd.dart (hunk 5 above)
Comment (kevmoo) [reply bin/dhttpd.dart]: ack

File: .travis.yml (hunk 1 above)
Comment (kevmoo) [reply .travis.yml]: Known issue 😢

**Pull request merged**